### ISSUE TITLE
question about paper D1,D2,D3

### ISSUE THREAD

**Issue by ANABUR920** [prasoongoyal/rl-lang-action-reward-network#1]
nice work, guys. I want to follow your work.
If I want to reproduce the baseline, and what is the specific dimension for D1, D2, and D3 in paper  Neural Network Details session.

**Comment by prasoongoyal** [comment]
The default values of the arguments should work. The algorithm was not sensitive to most of the hyperparameters.
